controllers/alertsconfig_controller.go

```diff
@@ -289,6 +289,7 @@ func (r *AlertsConfigReconciler) PatchIndividualAlertsConfigError(ctx context.Co
 	log = log.WithValues("alertsConfig_cr", alertsConfig.Name, "namespace", alertsConfig.Namespace)
 	alertStatus := alertsConfig.Status.AlertsStatus[alertName]
 	alertStatus.State = state
+	alertStatus.ErrorDescription = err.Error()
```

Comment:
Do you need to update errorDescription [here](https://github.com/keikoproj/alert-manager/blob/72b7cf458275d2f3bb853c989f9c53709fadcc4e/controllers/wavefrontalert_controller.go#L291) as well? I do not see this function being used anywhere though

Reply:
Wavefront controller is lil bit disoriented now- we should refactor that in future- but anyway i will update there too so that when we refactor it to use this function we will not have any issues